bukuserver/templates/bukuserver/lib.html

```diff
@@ -1,5 +1,11 @@
 {% macro filter(name, value) %}{{ url_for('bookmark.index_view', **{'flt0_'+name: value}) }}{% endmacro %}
 
+{% macro bookmarklet() -%}
+  {% with url = url_for('bookmarklet', _external=True) -%}
+    {% include 'bukuserver/bookmarklet.url' %}
+  {%- endwith %}
+{%- endmacro %}
+
```

Comment:
Injected with a macro (which supplies the base search URL); all extra whitespaces are removed.